### PR TITLE
aline opdb with cuda

### PR DESCRIPTION
**Pull request by yuchengliu1**
Some cuda bias decorators are in opdb. This PR replace them with xpu.

### CODE REVIEW

File: test/xpu/xpu_test_utils.py [hunk 1]
```diff
@@ -143,10 +151,6 @@
     "addr",
     "cdist",
     "nn.functional.group_norm",
-    "nn.functional.batch_norm",
-    "native_batch_norm",
-    "_native_batch_norm_legit",
-    "_batch_norm_with_update",
```
Comment (fengyuan14) [inline test/xpu/xpu_test_utils.py:149]: Why skip them? They are supported operators listed in op_db?
image

File: test/xpu/run_test_with_skip.py [hunk 2]
```diff
@@ -246,32 +243,67 @@ def launch_test(test_case, skip_list=None, exe_list=None):
...
-    "test_python_ref_errors__refs_where_xpu",  # align with CUDA, AssertionError: "Expected all tensors to be on the same device" does not match "Tensor on device xpu:0 is not on the expected device cpu!"
     "test_python_ref_executor__refs_pow_executor_aten_xpu_complex32",  # Didn't align with CUDA, Unexpected success
     "test_compare_cpu_nn_functional_grid_sample_xpu_float32",  # AssertionError: Tensor-likes are not close!
     "test_dtypes_nn_functional_batch_norm_without_cudnn_xpu",  # AssertionError: The supported dtypes for nn.functional.batch_norm on device type xpu are incorrect!
-    "test_out_native_batch_norm_xpu_float32", # CUDA XFAIL, The generated sample data does not meet the requirements.
-    "test_out__native_batch_norm_legit_xpu_float32", # CUDA XFAIL, The generated sample data does not meet the requirements.
-    "test_dtypes__batch_norm_with_update_xpu", # We are same as CUDA implementation. And CUDA skips these cases.
 
+    # AssertionError: Tensor-likes are not close!
+    "test_compare_cpu__refs_std_mean_xpu_float32",
```
Comment (fengyuan14) [inline test/xpu/run_test_with_skip.py:264]: Why we passed before,
image
But fail here?

File: test/xpu/run_test_with_skip.py [hunk 3]
```diff
@@ -1387,6 +1406,8 @@ def launch_test(test_case, skip_list=None, exe_list=None):
 res += launch_test("test_indexing_xpu.py", skip_list)
 
 # test_pooling
+
+
```
Comment (fengyuan14) [inline test/xpu/run_test_with_skip.py:1410]:
```suggestion
```

File: test/xpu/xpu_test_utils.py (hunk 1 above)
Comment (yuchengliu1) [reply test/xpu/xpu_test_utils.py]: revert it

File: test/xpu/run_test_with_skip.py (hunk 2 above)
Comment (yuchengliu1) [reply test/xpu/run_test_with_skip.py]: local result is different with pre-ci result. take the pre-ci result and revert the skip list

**Review by fengyuan14 — APPROVED** [review]

File: test/xpu/run_test_with_skip.py [hunk 4]
```diff
@@ -2929,61 +2912,77 @@ def launch_test(test_case, skip_list=None, exe_list=None):
 res += launch_test("nn/test_pruning_xpu.py")
 
 skip_list = (
+    # Unexpected success
```
Comment (fengyuan14) [inline test/xpu/run_test_with_skip.py:2915]:
```suggestion
    # CPU fallback fails. Implementation difference between CPU and CUDA. Expect success on CPU and expect fail on CUDA. When we use CPU fallback and align expected fail list with CUDA, these cases fail.
    # Unexpected success
```

**Pull request merged**